dev-docs/examples/in-renderer-integration.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: example
+title: in-renderer Integration Example
+description: in-renderer Integration (formerly known as “outstream”) Example
+
+sidebarType: 1
+
+about:
+  - in-renderer Integration is one of the integration methods for Prebid Video
+  - In this example, to keep it simple, ad rendering is performed using `pbjs.renderAd`
+  - InRenderer.js, an open-source renderer optimized for in-renderer integration, is used as the renderer for in-renderer integration. For more information, see <a href="https://github.com/hogekai/in-renderer-js" target="_blank">InRenderer.js documentation</a>
+---
+
+## Example
+
+{% capture htmlCodePrebid %}
+<script src="https://cdn.michao-ssp.com/original/prebid.js"></script>
```

Comment:
I would leave this PR open until https://github.com/prebid/Prebid.js/pull/12507 is merged and it can be replaced with "not-for-prod" distribution, so you don't have to maintain this distribution